docs/content/en/docs/clustermgmt/observability/cluster-verify.md

```diff
@@ -9,117 +9,36 @@ description: >
-eksa-system                         eksa-controller-manager-76f484bd5b-x6qld                         1/1     Running   0             3d22h
-etcdadm-bootstrap-provider-system   etcdadm-bootstrap-provider-controller-manager-6bcdd4f5d7-wvqw8   1/1     Running   0             3d22h
-etcdadm-controller-system           etcdadm-controller-controller-manager-6f96f5d594-kqnfw           1/1     Running   0             3d22h
-kube-system                         cilium-lbqdt                                                     1/1     Running   0             3d22h
-kube-system                         cilium-operator-55c4778776-jvrnh                                 1/1     Running   0             3d22h
-kube-system                         cilium-operator-55c4778776-wjjrk                                 1/1     Running   0             3d22h
-kube-system                         cilium-psqm2                                                     1/1     Running   0             3d22h
-kube-system                         coredns-69797695c4-kdtjc                                         1/1     Running   0             3d22h
-kube-system                         coredns-69797695c4-r25vv                                         1/1     Running   0             3d22h
-kube-system                         etcd-mgmt-clrt4                                                  1/1     Running   0             3d22h
-kube-system                         kube-apiserver-mgmt-clrt4                                        1/1     Running   0             3d22h
-kube-system                         kube-controller-manager-mgmt-clrt4                               1/1     Running   0             3d22h
-kube-system                         kube-proxy-588gj                                                 1/1     Running   0             3d22h
-kube-system                         kube-proxy-hrksw                                                 1/1     Running   0             3d22h
-kube-system                         kube-scheduler-mgmt-clrt4                                        1/1     Running   0             3d22h
-kube-system                         kube-vip-mgmt-clrt4                                              1/1     Running   0             3d22h
-kube-system                         vsphere-cloud-controller-manager-7vzjx                           1/1     Running   0             3d22h
-kube-system                         vsphere-cloud-controller-manager-cqfs5                           1/1     Running   0             3d22h
-kube-system                         vsphere-csi-controller-79fb7fbb6b-9vvr7                          5/5     Running   0             3d22h
-kube-system                         vsphere-csi-node-bths4                                           3/3     Running   0             3d22h
-kube-system                         vsphere-csi-node-r8lwp                                           3/3     Running   0             3d22h
+kubectl get pods -n kube-system | head -n 5
```

Comment:
Does `get pods` always have the same sorting? Maybe grep for the pods you want to show instead of head?

Reply:
Hi @TerryHowe , Here we are just checking if the pods are visible or not, there need not to be any specific order here.